doc/source/train/user_guide.rst

```diff
@@ -1240,3 +1240,20 @@ Backwards Compatibility with Ray SGD
 ------------------------------------
 
 If you are currently using :ref:`RaySGD <sgd-index>`, you can migrate to Ray Train by following: :ref:`sgd-migration`.
+
+FAQ
```

Comment:
Yeah, certainly. I had originally decided against creating a separate page in this PR since I wasn't sure how the reader would react if there was just one question listed.

Are there any sections in the existing Train User Guide that you feel should be converted to FAQ? Or any other questions you think are worth answering to start it off?